docs/interactions/Receiving_and_Responding.md

```diff
@@ -38,21 +38,22 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
-
+| Field           | Type                                                                                                                                                                                                                         | Description                                                                                                                                                                           | Interaction Type                                                                                                                                   |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id              | snowflake                                                                                                                                                                                                                    | the [`ID`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                   | Application Command, Modal                                                                                                                         |
+| name            | string                                                                                                                                                                                                                       | the [`name`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command, Modal                                                                                                                         |
+| type            | integer                                                                                                                                                                                                                      | the [`type`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command, Modal                                                                                                                         |
```

Comment:
```suggestion
| id              | snowflake                                                                                                                                                                                                                    | the [`ID`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                   | Application Command                                                                                                                                |
| name            | string                                                                                                                                                                                                                       | the [`name`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command                                                                                                                                |
| type            | integer                                                                                                                                                                                                                      | the [`type`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command                                                                                                                                |
```
These fields aren't used in `MODAL` interaction (neither in `MODAL_SUBMIT`), also, I think `title` field is missing here.

Reply:
I'm not sure about title, could you send me the complete payload?

I remember it being in data, but I'm completely brain dead this weekend.

Reply:
It's used by `MODAL` interaction to set the modal title.

Reply:
it's in data

Reply:
i don't think that you get it back 🤔

Reply:
> also, I think title field is missing here.

By "here" I meant in Interaction Data Structure table.

Reply:
> i don't think that you get it back 🤔

No, you don't, but as I said, it's used to set the modal's title.

Reply:
Marking as resolved, was a missunderstanding